spec/ddtrace/contrib/action_mailer/helpers.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails'
+
+RSpec.shared_context 'ActionMailer helpers' do
+  before(:each) do
+    if ActionMailer::Base.respond_to?(:delivery_method)
```

Comment:
Is this condition for? Different version of Rails?

Reply:
added a comment